openhands/runtime/client/runtime.py

```diff
@@ -317,9 +316,9 @@ def _wait_until_alive(self):
         if not self.log_buffer.client_ready:
             time.sleep(1)
             attempts = 0
-            while not self.log_buffer.client_ready and attempts < 4:
+            while not self.log_buffer.client_ready and attempts < 5:
```

Comment:
What is the reason for more attempts? It's a bit surprising, because if it is related to this issue, it seems more attempts would make it worse, since it keeps trying and trying to get logs?

Reply:
Overall shorter: the sleep is now 1 second, not 2 :)

Reply:
Also, this section is only run once during client initialization.